src/PrestaShopBundle/Resources/config/routing/admin/improve/design/_mail_theme_alias.yml

```diff
@@ -0,0 +1,22 @@
+# These are alias routes used to manage legacy urls that were badly formatted (camel case)
+_admin_mail_theme_save_configuration_alias:
```

Comment:
Warning because of this behaviour https://github.com/PrestaShop/docs/issues/282
The `_` need to be removed!!